pom.xml

```diff
@@ -22,24 +22,23 @@
         <Implementation-Vendor>${project.organization.name}</Implementation-Vendor>
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
 
         <jetty.version>9.3.9.v20160517</jetty.version>
-        <jetty.plugin.version>9.2.3.v20140905</jetty.plugin.version>
 
-        <flow.version>6.0.9</flow.version>
-        <vaadin.version>19.0.8</vaadin.version>
-        <framework.version>8.12.2</framework.version>
+        <flow.version>23.0.1</flow.version>
+        <vaadin.version>${flow.version}</vaadin.version>
```

Comment:
nit: would it be better to align those the other way around: `<flow.version>${vaadin.version}</flow.version>`? 🤔

Reply:
ok, will do in next PR